x-pack/qa/security-example-spi-extension/build.gradle

```diff
@@ -25,6 +25,9 @@ testClusters.all {
 
   setting 'xpack.security.authc.realms.custom.custom.order', '0'
   setting 'xpack.security.authc.realms.custom.custom.filtered_setting', 'should be filtered'
+  setting 'xpack.security.authc.realms.custom.custom.username', 'test_user'
+  keystore 'xpack.security.authc.realms.custom.custom.password', 'secret_password'
```

Comment:
This is not new in this PR. But I find the repetition of `custom` confusing as well. Probably better to have something like `custom.custom1` or `custom.default_custom`.